OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -689,6 +690,30 @@ static bool Name(S server, Connection conn, Session.Client client, string s)
 			return true;
 		}
 
+		static bool Handicap(S server, Connection conn, Session.Client client, string s)
+		{
+			var parts = s.Split(' ');
+			var targetClient = server.LobbyInfo.ClientWithIndex(Exts.ParseIntegerInvariant(parts[0]));
+
+			// Only the host can change other client's info
+			if (targetClient.Index != client.Index && !client.IsAdmin)
+				return true;
+
+			int handicap;
+			if (!Exts.TryParseIntegerInvariant(parts[1], out handicap) && handicap >= 50 && handicap <= 100)
```

Comment:
this would then be matching against the levels defined on the trait - see the faction or bot commands for reference on how to do this.